conf/test_manifest.config

```diff
@@ -0,0 +1,25 @@
+/*
+ * -------------------------------------------------
+ *  Nextflow config file for running tests
+ * -------------------------------------------------
+ * Defines bundled input files and everything required
+ * to run a fast and simple test. Use as follows:
+ *   nextflow run nf-core/ampliseq -profile test_manifest
```

Comment:
is it on purpose that this in not specified in `.github/workflows/ci.yml`?

Reply:
Uff, good catch. I hope this is actually working! This really needs to be done! I'll do a PR to dev once more...